src/PageNotFound/PageNotfound.tsx

```diff
@@ -0,0 +1,18 @@
+import { Container, Grid } from '@material-ui/core'
+import React from 'react'
+import  PNFImage from '../Media/PageNotFoundImage.jpg'
+
+interface Props {}
+
+function PageNotfound() {
+	return (
+		<Grid container alignItems="center" justify="center">
+			<Grid item>
+				<h3>Page not found. How did you even get here?</h3>
+			</Grid>
+            <img src={PNFImage} width='300' height='400'/>
+		</Grid>
```

Comment:
Indentation was off here; also the attributes should use double quotes =]

```suggestion
		<Grid container alignItems="center" justify="center">
			<Grid item>
				<h3>Page not found. How did you even get here?</h3>
			</Grid>
			<img src={PNFImage} width="300" height="400" />
		</Grid>
```